src/docs/development/add-to-app/ios/project-setup.md

```diff
@@ -221,6 +221,38 @@ some/path/MyApp/
   the flags `--xcframework --no-universal`.
 {{site.alert.end}}
 
+If you don't want distribuite Flutter.framework for other developer or machines, you can generate Flutter as `pubspec` adding the flag `--cocoapods`. Will produce a `Flutter.podspec` instead of an engine Flutter.framework (recomended if host app uses CocoaPods).
+
+```sh
+flutter build ios-framework --cocoapods --output=some/path/MyApp/Flutter/
+```
+
```

Comment:
As @xster points out in https://github.com/flutter/flutter/issues/47696#issuecomment-571228682 we can merge this to master now as long as we have a caveat the flag is available on v1.13.6+.

```
{{site.alert.important}}
  The `--cocoapods` flag is available in Flutter v1.13.6.
{{site.alert.end}}
```